python/ray/services.py

```diff
@@ -957,12 +957,72 @@ def start_raylet(redis_address,
     return raylet_name
 
 
+def determine_plasma_store_config(object_store_memory=None,
+                                  plasma_directory=None,
+                                  huge_pages=False):
+    """Figure out how to configure the plasma object store.
+
+    Args:
+        object_store_memory: The user-specified object store memory parameter.
+        plasma_directory: The user-specified plasma directory parameter.
+        huge_pages: The user-specified huge pages parameter.
```

Comment:
Types?

Reply:
Fixed. However, we haven't been consistent about types. Do you think we should use them or not? Do they add value?